test/mockSchema.ts

```diff
@@ -337,7 +337,7 @@ export const mockSchema: (ERC725JSONSchema & {
     ]),
     returnGraphData:
```

Comment:
We don't support graphdata anymore, no need to maintain this, you can simply delete.

Reply:
@Hugoo 
We need to keep this change otherwise the tests fail. As now we use `hexToNumber` in `encoder.ts` to decode `uintN` numbers.

This function used to return numbers as `string`, but now returns numbers as `number`.
See below when testing on the console of universalprofile.cloud

<img width="616" alt="image" src="https://github.com/ERC725Alliance/erc725.js/assets/31145285/70dfd536-7331-4292-bb49-18221b115d27">